main.md

```diff
@@ -27,6 +27,9 @@ Git supports the notion of a “shallow clone”, which is a more succinctly mea
 
 `$ git log --oneline`
  - Shows history of past commits in summary which contains only commit id and commit message. 
+ 
+ `$ git blame <file>`
+ - Shows the last person who have changed the line in a document upto the last commit.
```

Comment:
" Shows what revision and author last modified each line of a file upto the last commit. "
Please change the description to this.